.github/actions/publish-images/action.yml

```diff
@@ -35,7 +35,7 @@ runs:
     - name: "Test code"
       if: inputs.run_tests == 'true'
       shell: bash
-      run: ./gradlew check
+      run: ./gradlew check -PGITHUB_ACCESS_TOKEN=${{ inputs.access_token }}
```

Comment:
To address [this error](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/4000415259/jobs/6865654740#step:5:621).